src/lib/LUA/rx_devLUA.cpp

```diff
@@ -535,10 +535,6 @@ static void registerLuaParameters()
 
 eSerialProtocol prot0 = config.GetSerialProtocol();
 bool hasMavlink = prot0 == PROTOCOL_MAVLINK;
-#if defined(PLATFORM_ESP32)
-  eSerial1Protocol prot1 = config.GetSerial1Protocol();
-  hasMavlink = hasMavlink || (prot1 == PROTOCOL_MAVLINK);
-#endif
 
   if (hasMavlink)
```

Comment:
Actually, these should always be registered i.e. no if block.
And at the end of event loop the fields should be shown/hidden depending on the protocol selected, similar to the backpack options on the TX side of things.